protobuf/psa_generate_random.proto

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2020 Contributors to the Parsec project.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package psa_generate_random;
+
+import "psa_algorithm.proto";
```

Comment:
I don't think you need this import here!

Reply:
Quite right, removed it.